samples/v1p1beta1/receiveNotifications.js

```diff
@@ -0,0 +1,84 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+'use strict';
+
+function main(
+    projectId = 'your-project-id', subscriptionId = 'your-subscription-id') {
+
+  // [START scc_receive_notifications]
+  const {PubSub} = require('@google-cloud/pubsub');
+  const {StringDecoder} = require('string_decoder');
+
+  const gax = require('google-gax');
+  const path = require('path');
+  const protobuf = require('protobufjs');
+
+  let protoFilesRoot = new gax.GoogleProtoFilesRoot();
+  // Load in proto root for Finding.
+  protoFilesRoot = protobuf.loadSync(
```

Comment:
We generally don't want to expose this low level of an interface to folks.  Can you tell us a little about what you're trying to do?  cc @alexander-fenster

Reply:
What I'm attempting to do here is "importing" these 2 protos so that they can be created later on in the file. I'm unfamiliar with NodeJS (or JS as a whole), so I was following some code that a teammate had written earlier.

Is there a better way to import the proto? Or do I even need to import the proto and can just call JSON.parse on it later?

Reply:
Calling for help from @bcoe  :)

Reply:
I don't understand why we would need to import a proto, the point of the client libraries is to provide an abstraction on top of the grpc and protobuffer layer.

I would expect this to be something along the lines of:

```js
const {
    SecurityCenterClient,
} = require('@google-cloud/security-center').v1p1beta1;
const client = new SecurityCenterClient();
client.createFinding();
```

see: https://github.com/googleapis/nodejs-security-center/blob/master/src/v1p1beta1/security_center_client.ts#L455

Reply:
When I say create, I only mean on the client side so the user has an object that they can manipulate how they please. In this file, we receive a pubsub message in the form of a bytestring of JSON. I decoded this data into a NotificationMessage proto object. Is it better to just leave it as JSON?

Reply:
@tdh911 I think for the JavaScript audience, it would be better to leave it a JSON object. If we eventually want a more complex type, I think we'd do better to expose this explicitly in the generated code. CC: @alexander-fenster.

Reply:
Samples are JavaScript, there is no need to import any proto files. Just create a plain JSON object with the needed fields. You can consult `protos/protos.d.ts` for a list of fields and types of that object, but normally, it's just something like

```js
const request = { field1: value, field2: value };
```

Unless it's some very special case, there should be no need to use `ProtoFilesRoot` in samples.

Reply:
Got it, switched over to just leaving it as a JSON object.